data-raw/simulate_longitudinal_data.R

```diff
@@ -213,7 +203,7 @@ for (i in 1:12) {
                             dfParticipants = dfParticipants)
   dfQUERY <- AddQUERY(df = dfQUERY,
                       dfParticipants = dfParticipants)
-  dfDATACHG <- AddDATACHG(df = dfSTUDCOMP,
+  dfDATACHG <- AddDATACHG(df = dfDATACHG,
```

Comment:
@zdz2101 I found this typo, so i think simulation needs to be run again. it was existing prior to this PR.